Week-2/Week2Assignment1/styles.css

```diff
@@ -0,0 +1,230 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Nunito", sans-serif;
+  background-color: #222222;
+  color: white;
+}
+
+body {
+  width: 95vw;
+}
+
+.heading {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  margin-top: 20px;
+}
+
+.heading > h1 {
+  margin-bottom: 10px;
+}
+
+.heading > p {
+  font-size: 1.2rem;
+  font-weight: 600;
+}
+
+.section {
+  display: flex;
```

Comment:
instead of writing redundant code for display: flex, make a class only with flex property and use that everywhere.

Reply:
Same goes for width: 100%